### PR TITLE
Improve world generator diagnosis

### PR DESCRIPTION
**Pull request by N0tMyFaultOG**
## Overview

**Fixes https://github.com/IntellectualSites/PlotSquaredSuggestions/issues/62**

## Description
Can be obtained from the suggestion

## Checklist

- [X] I included all information required in the sections above
- [X] I tested my changes and approved their functionality
- [X] I ensured my changes do not break other parts of the code
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/PlotSquared/blob/breaking/CONTRIBUTING.md)

### CODE REVIEW

File: Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java [hunk 1]
```diff
@@ -115,6 +115,15 @@ private static String readFile(@NonNull final File file) throws IOException {
                         "&cSkipping PlotSquared.use_THIS.yml because it's empty");
                 }
 
+                try {
```
Comment (Sauilitired) [inline Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: The command description should be altered to mention this file as well, to not cause any further issues with people claiming we steal their data. Last time wasn't very fun.

File: Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java [hunk 2]
```diff
@@ -115,6 +115,15 @@ private static String readFile(@NonNull final File file) throws IOException {
...
+                    final File MultiverseWorlds =
+                            new File(PlotSquared.get().IMP.getDirectory(), "../../plugins/Multiverse-Core/worlds.yml");
```
Comment (Sauilitired) [inline Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: Why is this going back two folders, to then re-enter plugins?

Comment (N0tMyFaultOG) [reply Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: Having `/plugins/Multiverse-Core/worlds.yml` does not lead to the expected result

Comment (MattBDev) [reply Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: Getting in certain directories with Bukkit has always been a pain.

Comment (N0tMyFaultOG) [reply Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: I had no luck doing it in a better way, so I used that. If you have a better AND working solution, hmu @Sauilitired :^)

Comment (Sauilitired) [reply Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: Lol that wasn’t my point, but if you go backwards two steps you’re already leaving the plugin folder, then you directly re-enter it. What would be different by just doing ../Multiverse-Core?

Comment (N0tMyFaultOG) [reply Core/src/main/java/com/github/intellectualsites/plotsquared/plot/commands/DebugPaste.java]: That works! I was not entirely sure how the paths behaved so I just wanted to be sure it works ^^

**Pull request merged**